metadata-builders/src/main/java/com/linkedin/metadata/builders/search/DataFlowIndexBuilder.java

```diff
@@ -0,0 +1,88 @@
+package com.linkedin.metadata.builders.search;
+
+import com.linkedin.common.Ownership;
+import com.linkedin.common.urn.DataFlowUrn;
+import com.linkedin.datajob.DataFlowInfo;
+import com.linkedin.data.template.RecordTemplate;
+import com.linkedin.data.template.StringArray;
+import com.linkedin.metadata.search.DataFlowDocument;
+import com.linkedin.metadata.snapshot.DataFlowSnapshot;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import javax.annotation.Nonnull;
+import lombok.extern.slf4j.Slf4j;
+
+
+@Slf4j
+public class DataFlowIndexBuilder extends BaseIndexBuilder<DataFlowDocument> {
+  public DataFlowIndexBuilder() {
+    super(Collections.singletonList(DataFlowSnapshot.class), DataFlowDocument.class);
+  }
+
+  @Nonnull
+  private static String buildBrowsePath(@Nonnull DataFlowUrn urn) {
+    return ("/" + urn.getOrchestratorEntity() + "/"  + urn.getFlowIdEntity() + "/"  + urn.getClusterEntity()).toLowerCase();
```

Comment:
wouldn't we want this to be `getOrchestratorEntity / getClusterEntity / getFlowIdEntity `?

Reply:
Makes perfect sense!